GoogleDataTransportCCTSupport/GDTCCTLibrary/GDTCCTNanopbHelpers.m

```diff
@@ -148,7 +148,13 @@ gdt_cct_LogEvent GDTCCTConstructLogEvent(GDTCOREvent *event) {
     logEvent.has_network_connection_info = 1;
   }
   NSError *error;
-  NSData *extensionBytes = [NSData dataWithContentsOfURL:event.fileURL options:0 error:&error];
+  NSData *extensionBytes;
+  if (event.fileURL) {
+    extensionBytes = [NSData dataWithContentsOfURL:event.fileURL options:0 error:&error];
+  } else {
+    GDTCORLogError(GDTCORMCEFileWriteError, @"%@", @"An event's fileURL property was nil.");
```

Comment:
Should be `GDTCORMCEFileReadError ` I guess

Reply:
Of course, you're right. Ty.